src/app.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import List from './list.js';
+
+class App extends Component {
+    constructor() {
+        super();
+        const todos = JSON.parse(localStorage.getItem("todos"));
+        this.state = {
+            todos: todos || [],
+            newItem: {
+                text: '',
+                completed: false
+            }
+        }
+        this.handleNewToDo = this.handleNewToDo.bind(this);
+        this.addItem = this.addItem.bind(this);
+        this.handleClick = this.handleClick.bind(this);
+        this.removeItems = this.removeItems.bind(this);
+    }
+    addItem(event) {
+        event.preventDefault();
+        const todos = this.state.todos;
+        const item = this.state.newItem
+        todos.push(item);
+        this.setState({
+            todos: todos,
+            newItem: {
+                text: '',
+                completed: false
+            }
+        })
+        localStorage.setItem("todos", JSON.stringify(todos));
```

Comment:
I like that you are using localStorage to persist some data. But Component state is just fine for a todo list built with the client.